cms/settings/base.py

```diff
@@ -196,38 +197,51 @@
 #
 # Do not use the same Redis instance for other things like Celery!
 
-# Prefer the TLS connection URL over non
-REDIS_URL = env.get("REDIS_TLS_URL", env.get("REDIS_URL"))
+CACHES: dict = {"memory": {"BACKEND": "django.core.cache.backends.locmem.LocMemCache", "LOCATION": "memory"}}
 
-if REDIS_URL:
-    connection_pool_kwargs: dict[str, str | None] = {}
+redis_options = {
+    "IGNORE_EXCEPTIONS": True,
+    "SOCKET_CONNECT_TIMEOUT": 2,  # seconds
+    "SOCKET_TIMEOUT": 2,  # seconds
```

Comment:
The timeouts are simply to ensure that if there are network issues with the cache, skip it, as opposed to waiting, since it'd replace the benefits of having a cache in the first place.